ansible/playbooks/start-redpanda.yml

```diff
@@ -45,6 +46,16 @@
         --ips {{ groups["redpanda"] | map('extract', hostvars) | map(attribute='private_ip') | join(',') }}
       {% endif %}
 
+      {% if rack is defined and rack != -1 %}
+      sudo rpk redpanda config set redpanda.rack {{ rack }}
+      {% endif %}
+
+  - name: enable rack awareness
+    shell: |
+      set -e
+      rpk cluster config set enable_rack_awareness true --api-urls {{ hostvars[inventory_hostname].private_ip }}:9644
+
+    when: rack is defined and rack != -1
```

Comment:
[Suggestion] what do you think of leaving these 2 together:

```yaml
  - name: enable rack awareness
    shell: |
      set -e
      sudo rpk redpanda config set redpanda.rack {{ rack }}
      rpk cluster config set enable_rack_awareness true --api-urls {{ hostvars[inventory_hostname].private_ip }}:9644
    when: rack is defined and rack != -1
```

Reply:
@vuldin has addressed this

Reply:
Now resolved on main branch